### DISCUSSION TITLE
Improve `python.analysis.completeFunctionParens`

### DISCUSSION THREAD

**Discussion by byehack** [microsoft/pylance-release#3541] [Enhancement]
Enable `python.analysis.completeFunctionParens`
For what I expected for this feature See Examples below: (`|` as cursor)

## Ex1
```py3
a = |"123456"
```
type `int` then accept suggestion:
```py3
a = int(|"123456")   # also it can expand selection in parentheses
```

## Ex2
```py3
a = |"123456" and "789"
```
type `int` then accept suggestion:
```py3
a = int(|"123456" and "789")   # also it can expand selection in parentheses
```

## Ex3
```py3
a = |("123456" and "789")
```
type `int` then accept suggestion:
```py3
a = int(|"123456" and "789")   # also it can expand selection in parentheses
```

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
